status-go-version.json

```diff
@@ -2,7 +2,7 @@
     "_comment": "DO NOT EDIT THIS FILE BY HAND. USE 'scripts/update-status-go.sh <tag>' instead",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.62.3",
-    "commit-sha1": "397ee840c650b7fd4a414b38e200274d2d1b716c",
-    "src-sha256": "1x6cs06qqbncvlbkw2ch3alkrhf24m966c7jkvvz6rb9fhbxgrgj"
+    "version": "v0.62.7",
```

Comment:
actually are you sure we can use this version? or should be make a branch from 0.62.3 and add only your changes for hotfix and make hotfix version ?

Reply:
I think is probably best if we use HEAD, it contains a few more commits, but because this touches migrations, we risk having them out-of-order, so it's probably less risky to have HEAD (it's 4 extra commits, already tested).

Reply:
already tested in hotfix? will they work with outdated status-react ?

Reply:
Migrate db to shorter path name looks scary for hotfix

Reply:
Yes, it does look scary, though has been tested, let's create an hotfix branch :)